packages/components/table/src/table-body/events-helper.ts

```diff
@@ -102,7 +102,7 @@ function useEvents<T>(props: Partial<TableBodyProps<T>>) {
      *    - Expected: 188
      *    - Actual: 188.00000762939453
      */
-    const rangeWidth = Math.round(range.getBoundingClientRect().width)
+    const rangeWidth = Math.ceil(range.getBoundingClientRect().width)
```

Comment:
This may cause problems fixed by #10816.

`Math.ceil` and `Math.round` may not work here. For the problem solved by #10816, it may be roughly judged, for example, only 5 digits after the decimal point are used to solve it.
```ts
const rangeWidth = Number(range.getBoundingClientRect().width.toFixed(5))
```
This may not be entirely accurate though, do you guys have any good workarounds? @holazz @Cenavy1019

Reply:
I think it's okay